packages/api/src/util/ensemble.js

```diff
@@ -27,9 +27,14 @@ if (process.env.NODE_ENV === 'integration') {
   baseUrl = 'https://api.beta.litefarm.org';
 } else if (process.env.NODE_ENV === 'production') {
   baseUrl = 'https://api.app.litefarm.org';
+} else if (process.env.NODE_ENV === 'development') {
```

Comment:
As far as I know `process.env.NODE_ENV` is always defaulting to `development` during local development (e.g. if you run `npm run nodemon`). We should probably have some sort of separate mode which we only set if the ngrok commands are run

Reply:
The github tests set the NODE_ENV to 'pipeline'. I think in this case its fine since the webhook will always have an error unless we use ngrok or something of the sort. I also don't think we want to have a separate mode for ngrok since almost everything will be the same as development when using ngrok. I think the only the difference would be the api URL used in to make calls from the frontend.